src/templates/en/2019/base.html

```diff
@@ -140,6 +140,7 @@
 {% block appendices %}Appendices{% endblock %}
 
 {% block ebook_title %}Ebook{% endblock %}
+{% block ebook_download_short %}Ebook PDF ({{ ebook_size_in_mb }}MB){% endblock %}
```

Comment:
Nit: Should we try to use this pattern more to avoid retranslations for the same phrases?
```suggestion
{% block ebook_download_short %}{{ ebook_title }} PDF ({{ ebook_size_in_mb }}MB){% endblock %}
```

Reply:
That actually becomes:

```
{% block ebook_download_short %}{{ self.ebook_title() }} PDF ({{ ebook_size_in_mb }}MB){% endblock %}
```

I'm torn on this myself. Yes it's reuse but a bit more confusing for translators to read and ensure they understand and translate appropriately (e.g. change ordering to "PDF Ebook" for some languages).

Think clearer to leave as is to be honest.